### PR TITLE
python-pattern and hello word

### PR DESCRIPTION
**Pull request by athul4554**
This contribution consist of pattern making with python and Hello world program with python

### CODE REVIEW

**Comment by aryangulati** [comment]
hey @athul4554 Projects/python_patterns/hello_world_athul.py this has been added already in hello World and please add some other hello file instead of hello_world.py in the hello-world folder and please read the guidelines ones

**Review by Zircoz — CHANGES_REQUESTED** [review]
Hello world is already added

File: Projects/python_patterns/hello_world_athul.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# LANGUAGE: Python
```
Comment (Zircoz) [inline Projects/python_patterns/hello_world_athul.py:1]: This file not needed

File: Projects/python_patterns/menspants.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+rows = 14
+print("*" * rows, end="\n")
+i = (rows // 2) - 1
+j = 2
+while i != 0:
+    while j <= (rows - 2):
+        print("*" * i, end="")
+        print("_" * j, end="")
```
Comment (Zircoz) [inline Projects/python_patterns/menspants.py:8]: Men's pant's legs aren't connected, right?
replace that underscore with white space

File: Projects/python_patterns/triangular_pattern.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+size = 7
+m = (2 * size) - 2
+for i in range(0, size):
+    for j in range(0, m):
+        print(end=" ")
+    m = m - 1  # decrementing m after each loop
+    for j in range(0, i + 1):
+        # printing full Triangle pyramid using stars
+        print("* ", end=' ')
```
Comment (Zircoz) [inline Projects/python_patterns/triangular_pattern.py:10]: It's not printing right enough due to that white space after asterisk, remove it

**Pull request closed without merging**